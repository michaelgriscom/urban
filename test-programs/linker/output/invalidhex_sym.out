Label          	Local Addr	Adjustment	Reloc. Addr	Module Len
InvalidHex     	         0	         0	          0	       256
