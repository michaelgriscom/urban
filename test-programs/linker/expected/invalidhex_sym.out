Symbol			Assembler Address		Linker Address		Length		Adjustment
InvalidHex		000000					000000				0100		0000
